<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonDeplacerTout">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Déplacer tout</string>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/images/deplacement.png</normaloff>:/images/deplacement.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNeRienFaire">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Ne rien faire pour tout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSupprimerTout">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Supprimer tout</string>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/images/annuler.png</normaloff>:/images/annuler.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>9</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Déplacer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ne rien faire</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date création</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date modification</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Taille</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Série présente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sous-dossier</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date création</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date modification</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Taille</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Déplacer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ne rien faire</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonDeplacerFichier">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Déplacer/supprimer les fichiers</string>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/images/deplacement.png</normaloff>:/images/deplacement.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRetour">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/images/retour.png</normaloff>:/images/retour.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
